proto/desmos/profiles/v2/models_relationships.proto

```diff
@@ -16,7 +16,10 @@ message Relationship {
 
   string creator = 1 [ (gogoproto.moretags) = "yaml:\"creator\"" ];
   string recipient = 2 [ (gogoproto.moretags) = "yaml:\"recipient\"" ];
-  string subspace = 3 [ (gogoproto.moretags) = "yaml:\"subspace\"" ];
+  string subspace_id = 3 [
+    (gogoproto.customname) = "SubspaceID",
+    (gogoproto.moretags) = "yaml:\"subspace\""
```

Comment:
```suggestion
    (gogoproto.moretags) = "yaml:\"subspace_id\""
```

Reply:
This comment (and all the ones below) actually refer to the Proto definitions for either `v1` or `v2`, which I think should be kept as they are rather than changed (even thought the change to the YAML tag does not have any effect since data is stored as bytes).